frontend/src/components/Project/ViewProject.tsx

```diff
@@ -0,0 +1,205 @@
+import * as React from 'react';
+import { DataGrid, GridColDef } from '@material-ui/data-grid';
+import { useEffect, useState } from 'react';
+import { getAllBusinessProject } from '../../services/ProjectAPI';
+import { Button, Grid, Link } from '@material-ui/core';
+import { useHistory } from 'react-router-dom';
+
+const ViewProject: React.FC = () => {
+  const [projectList, setProjectList] = useState<any>([]);
+  const history = useHistory();
+
+  interface Sale {
+    amount: number;
+  }
+
+  interface Data {
+    id: number;
+    title: string;
+    email: string;
+    description: string;
+    status: string;
+    serviceType: string;
+    createdDate: string;
+    followUpDate: string;
+    deadlineDate: string;
+    leadSource: string;
+    leadRanking: string;
+    leadCredit: string;
+    sale: Sale;
+  }
+  interface DataDisplay {
+    id: number;
+    title: string | any;
+    email: string;
+    description: string;
+    status: string;
+    service: string;
+    createdDate: string;
+    followUpDate: string;
+    deadlineDate: string;
+    leadSource: string;
+    leadRanking: string;
+    leadCredit: string;
+    sale: number;
+  }
+
+  const [select, setSelection] = useState<any>('');
+  const handleRowSelection = (id: any) => {
```

Comment:
is there a specific datatype for id? If it comes from html, I think it is a string.